kong/db/schema/init.lua

```diff
@@ -1681,7 +1681,7 @@ function Schema:process_auto_fields(data, context, nulls, opts)
         end
       end
 
-      if is_select and sdata.translate_backwards and opts.expand_shorthands then
+      if is_select and sdata.translate_backwards and opts and not(opts.hide_shorthands) then
```

Comment:
```suggestion
      if is_select and sdata.translate_backwards and opts and not opts.hide_shorthands then
```
`not` is a unary operator, no parentheses needed.